### PR TITLE
fix: Send back events related to the given contributor

### PR DESCRIPTION
**Pull request by Rossb0b**
related to https://github.com/NodeSecure/authors/issues/9

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -139,19 +139,27 @@ test("teardown", async() => {
...
-  const contributors = await github.getContributorsLastActivities("NodeSecure", "scanner");
+test("get contributor last activities", async(tape) => {
+  const [lastEvent] = await github.getContributorLastActivities({
+    owner: "NodeSecure",
+    repository: "scanner",
+    contributor: "fraxken"
+  });
 
-  tape.true(contributors.has("fraxken"));
+  tape.true(lastEvent.lastActivity);
```
Comment (tony-go) [inline test/test.js:149]: WDYT about having a more detailed assertion regarding what it is in `lastActivity`? not the values, but the properties or types? (or maybe you fear that octokit change their API too often)

**Review by Kawacrepe — APPROVED** [review]
lgtm

**Comment by fraxken** [comment]
@Kawacrepe @Rossb0b  Why do we need to implement Octokit ? Sure it's easier but it will introduce a lot of dependencies (59 to be exact).

**Comment by tony-go** [comment]
@fraxken raises a really good point.

To decide on this, we should answer the following question: how many APIs do we plan to add to this package? If it should not be updated too much, maybe we could stay on the current version. But if our future requirements will imply a lot of adding maybe it's good to have octo lib.

WDYT?

**Comment by tony-go** [comment]
We should discuss about this in the next meeting ^^

**Comment by fraxken** [comment]
The idea of github or gitlab repository is to be able to deal with the Github or Gitlab API in a lightweight way to download and extract GIT repositories (and tarballs/zip).

Also maybe those "github" API should be implemented directly in Authors (because anyway we will only use it here).

**Comment by Rossb0b** [comment]
I removed Octokit & fixed an issue with the usage of the GitHub token as env var.

But yeah, @fraxken is right, it probably should be in Authors directly.

**Comment by fraxken** [comment]
Closing PR. Please when you have time migrate the code to the opened PR in authors (and think to rebase).

**Comment by fraxken** [comment]
Link to the other PR: https://github.com/NodeSecure/authors/pull/17

**Pull request closed without merging**